language/testing-infra/e2e-tests/src/versioning.rs

```diff
@@ -3,10 +3,10 @@
 
 #![forbid(unsafe_code)]
 use crate::{account::Account, executor::FakeExecutor, utils};
-use diem_writeset_generator::old_releases::release_1_2_0_writeset;
+use diem_writeset_generator::old_releases::*;
 
 /// The current version numbers that e2e tests should be run against.
-pub const CURRENT_RELEASE_VERSIONS: &[u64] = &[1, 2];
+pub const CURRENT_RELEASE_VERSIONS: &[u64] = &[1, 2, 3];
```

Comment:
It seems a bit odd to have the versions here not match with the versions of the writesets that are being applied. Could you change it so that these numbers match with the version numbers of the release?